src/gettext_boost.cpp

```diff
@@ -11,6 +11,9 @@
 
    See the COPYING file for more details.
 */
+#if defined(_MSC_VER)
+#pragma warning(disable: 4714)
```

Comment:
Pretty sure it was _supposed_ to be just for `spirit_po` though, which is why it was there; so the mistake is probably forgetting the push/pop.